modules/ROOT/pages/execution-sequence-states-and-transactions.adoc

```diff
@@ -1,11 +1,9 @@
 = BPM process / task execution sequence, states, transitions
 :description: Understand the sequence of actions performed when a process instance is created and when a task is performed by a user
 
-Understand the sequence of actions performed when a process instance is created and when a task is performed by a user
+{description}
 
-This page also shows the interactions between the Bonita Portal and Bonita Engine.
-
-In the sequences described in this page, some of the actions apply only to *Teamwork*, *Efficiency*, *Performance*, or *Enterprise* editions, and are marked "Subscription only".
+This page also shows the interactions between the Bonita Applications and Bonita Engine.
```

Comment:
between the Bonita Applications => between Bonita Applications

Reply:
Added